egs/wsj/s5/steps/dur_model/make_nnet_config.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+learning_rate=0.001
+natural_gradient=false
+hidden_dim1=
+hidden_dim2=
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# != 2 ]; then
+   echo "Usage: $0 [options] <input-dim> <output-dim>"
+   echo "e.g.: $0 320 50"
+   exit 1;
+fi
+
+feat_dim=$1
+output_dim=$2
+
+if [ -z $hidden_dim1 ]; then
+  hidden_dim1=$[$feat_dim*3]
+fi
+
+if [ -z $hidden_dim2 ]; then
+  hidden_dim2=$[$output_dim*2]
+fi
+
```

Comment:
I just reduced the size of the second layer.
I will try an extra layer (3rd hidden layer) of low dimension right before softmax (no ReLU/Normalize layer after it).